src/pages/organize/[orgId]/campaigns/[campId]/index.tsx

```diff
@@ -29,13 +35,16 @@ export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
     await ctx.queryClient.prefetchQuery(['org', orgId], getOrg(orgId as string, ctx.apiFetch));
     const orgState = ctx.queryClient.getQueryState(['org', orgId]);
 
+    await ctx.queryClient.prefetchQuery(['people', orgId], getPeople(orgId as string, ctx.apiFetch));
+    const peopleState = ctx.queryClient.getQueryState(['people', orgId]);
```

Comment:
Is still needed now that you are searching using the search API instead of pre-fetching?